05_testing_and_ci/examples/python-testing/average_data.csv

```diff
@@ -0,0 +1,2 @@
+14,38,57,88,1,18,198
+59.14
```

Comment:
I purposely chose a data set which would give real valued mean so that I can use it to show how floating-point variables are tested. This is very much intentional and I would like to keep it.